libraries/chain/include/eosio/chain/config.hpp

```diff
@@ -114,7 +114,12 @@ const static uint32_t   setcode_ram_bytes_multiplier       = 10;     ///< multip
 
 const static uint32_t   hashing_checktime_block_size       = 10*1024;  /// call checktime from hashing intrinsic once per this number of bytes
 
+#ifdef __x86_64__
```

Comment:
This might be better checking for `EOSIO_EOS_VM_JIT_RUNTIME_ENABLED`

Reply:
Ah, definitely. I had forgotten we had those. Thanks for that.